File: src/main/java/br/com/leonardoferreira/jirareport/exception/ResourceNotFound.java
package br.com.leonardoferreira.jirareport.exception;

import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.ResponseStatus;

@ResponseStatus(value = HttpStatus.NOT_FOUND, reason = "Resource Not Found")
public class ResourceNotFound extends RuntimeException {
    private static final long serialVersionUID = 2093334930630431201L;

}

File: src/main/kotlin/br/com/jiratorio/exception/ResourceNotFound.kt
package br.com.jiratorio.exception

import org.springframework.http.HttpStatus
import org.springframework.web.bind.annotation.ResponseStatus

@ResponseStatus(value = HttpStatus.NOT_FOUND, reason = "Resource not found")
class ResourceNotFound : RuntimeException() {
    companion object {
        private val serialVersionUID = 2093334930630431201L
    }
}
